CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.7.2)
```

Comment:
Make should be added in separate PR fixing issue #21 
documentation should be also modified accordingly

Reply:
There was already another attempt in other branch https://github.com/libamtrack/library/blob/cmake_support/CMakeLists.txt

Reply:
This CmakeLists.txt for now is just for JavaScript because in configure you link to FORTRAN and that is why compile to JS fails. I dont get all libamtrack dependecies so can i get some help from you?

Reply:
Sure - I was planning to make my own CMake settings for libamtrack library. First step will be to deal with Fortran dependencies (i.e. replacing them with C version).
What is required from the side of emscripten ?

Reply:
The best would be use just GSL for  Landau
https://www.gnu.org/software/gsl/manual/html_node/The-Landau-Distribution.html
And then we can merge it to this branch and see if compile 
then i will try to compile root library to JS, because if it also contains some other language dependencie it will be point less. In the worst case scenerio we can make some "if" in Cmake to when compile to JS. What do you think?

Reply:
Moved to https://github.com/libamtrack/library/pull/30